website/content/en/docs/about/what-is-observability-pipelines.md

```diff
@@ -0,0 +1,20 @@
+---
+title: What is Observability Pipelines?
+weight: 2
+tags: ["concepts"]
+---
+
+Observability Pipelines is a commercial observability data pipeline for collecting, processing, and routing observability data from any source to any destination in the infrastructure that you own or manage.
```

Comment:
```suggestion
Observability Pipelines is a commercial observability data pipeline from Datadog for collecting, processing, and routing observability data from any source to any destination in the infrastructure that you own or manage.
```

I think this could make it clearer that this is a Datadog product.

Reply:
Thanks! I updated it to `Datadog's Observability Pipelines`.